frameworks/PHP/imi/imi-raw.dockerfile

```diff
@@ -5,18 +5,25 @@ RUN pecl install swoole > /dev/null && \
 
 RUN docker-php-ext-install pdo_mysql > /dev/null
 
+RUN pecl install redis > /dev/null && \
+    docker-php-ext-enable redis
+
 RUN apt -yqq update > /dev/null && \
     apt -yqq install git unzip > /dev/null
 
-WORKDIR /imi
+RUN apt -yqq install redis-server > /dev/null
```

Comment:
Is there a reason you're installing redis for tests that aren't doing cached queries?

Reply:
I have removed non-essential redis